src/components/InputFields.jsx

```diff
@@ -73,14 +75,15 @@ const InputFields = ({
 
     return (
         <div className="input-fields">
-            {isSettingsPage ? (
+            {isSettingsPage ? ( 
                 <>
                     <label>
                         Interest Rate (%):
                         <input
                             data-cy="interest-rate-input"
                             type="number"
                             value={
+
```

Comment:
Start adopting a practice where you give meaningful names to conditions. You could have a variable named `isInterestRateDefined = ....` and then use it here. It would be much more indicative for everyone who reads it and for future maintenance. Also, we wouldn't care to analyze what this line does OR why null and undefined were chosen here every time we read this line.